config/samples/flinkoperator_v1beta1_flinkjobcluster.yaml

```diff
@@ -21,6 +21,7 @@ spec:
   image:
     name: flink:1.14.2
   jobManager:
+    accessScope: Cluster
```

Comment:
Hi @JaeguKim Thanks for this! Which version are you using? I believe this was addressed and shouldn't be required now with the latest beta.

Reply:
I'm using [flink-cluster-crd.yaml](https://github.com/spotify/flink-on-k8s-operator/blob/master/helm-chart/flink-operator/templates/flink-cluster-crd.yaml) which is latest beta I believe.   
It looks like the [accessScope](https://github.com/spotify/flink-on-k8s-operator/blob/1967e49314bf15e4cc433c90ca51d1ae6114d5a2/helm-chart/flink-operator/templates/flink-cluster-crd.yaml#L3884) is still required field of `jobManager` property and [restartPolicy](https://github.com/spotify/flink-on-k8s-operator/blob/1967e49314bf15e4cc433c90ca51d1ae6114d5a2/helm-chart/flink-operator/templates/flink-cluster-crd.yaml#L1590) is also still required field of `job` property.

Reply:
ah I see what's going on, helm CRD is outdated.